docs/installation.rst

```diff
@@ -46,6 +46,19 @@ For the full configuration, please read :ref:`config`.
 
    Prior version **1.0.1** the extensions was called ``sphinxcontrib.needs``.
 
+.. _install_matplotlib_numpy:
+
+Matplotlib/NumPy support
+------------------------
+
+:ref:`needpie` and :ref:`needbar` uses `Matplotlib <https://matplotlib.org>`_ and `Numpy <https://numpy.org>`_ for generating graphs.
+
+The recommended install method (via `sphinxcontrib-needs[all]`) downloads
+matplotlib by default as well as all other optional extras.
```

Comment:
I used to say this when I wanted to remove `plantuml` too, but it's been made clear in separate PR this is a CORE feature (too bad, I don't need Java every day), so maybe I should update this text?

Reply:
I think you should just remove the segment "other optional extras" for now.